build.xml

```diff
@@ -182,7 +182,7 @@
 	<target name="compile" description="Compile the source files"
 		depends="checkBuildEnv,copyResources">
 		<javac srcdir="${src}" destdir="${classes}" debug="true"
-			deprecation="true" classpathref="src.classpath" target="1.6" source="1.6"
+			deprecation="true" classpathref="src.classpath" target="1.7" source="1.7"
```

Comment:
Jump to 1.8 directly? I think we agreed that once we moved past 1.6 we would not bother with 1.7 as an intermediary step.